stata_kernel/config.py

```diff
@@ -77,3 +69,10 @@ def raise_config_error(self, option):
         https://kylebarron.github.io/stata_kernel/user_guide/configuration/
         """.format(option)
         raise ValueError(dedent(msg))
+
+    def _remove_unsafe(self, key, permanent=False):
+        self.env.pop(key, None)
+        if permanent:
+            self.config.remove_option(option=key, section='stata_kernel')
+            with self.config_path.open('w') as f:
+                self.config.write(f)
```

Comment:
What does this do?

Reply:
Removes a config option. Namely I use it to remove graph_width and graph_height, which in effect resets them to their defaults.

As the function name implies, it is unsafe in that it does not check for whether the config should be removed. Careful internal use only!